common/test/acceptance/tests/studio/test_import_export.py

```diff
@@ -12,6 +12,25 @@
 
 
 # pylint: disable=no-member
+class TemplateTestMixin(object):
```

Comment:
@martynjames It grew naturally from the problem-- as the issue is that the template may change its text depending on whether or not a condition is satisfied, that meant that the issue **could** happen on the export page, too, and so it seemed sensible to add the test there, too, if a test needed to be added. Since both Libraries and Courses have this issue, and I didn't want to copy-paste code, this ended up being the result. I can try just placing a copy of the test in the main test mixin classes instead, even if they're a bit duplicated, if this will satisfy.